acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/models/Publication.java

```diff
@@ -99,5 +87,38 @@ public boolean isPubliclyAccessible() {
         return publiclyAccessible;
     }
 
+    public static class NominalPublicationDate {
```

Comment:
Should this be in its own file?

Reply:
Hmm, my reason for keeping `NominalPublicationDate` as a nested class of `Publication` was to emphasise that this class - and, especially, the logic that it encapsulates - only makes sense in the context of its parent class; contrary to the `RestrictableDate` in production, it was not meant to be a generic thing.

Nothing stops us from moving it out of this class but, for the reasons described above I'd prefer leaving it as is. Happy to have my mind changed, though :-)